src/site/content/ru/fast/remove-unused-code/index.md

```diff
@@ -0,0 +1,109 @@
+---
+layout: post-old
+title: Удаление неиспользуемого кода
+subhead: |
+  Npm упрощает добавление кода в проект. Но так уж ли нужны все эти лишние байты?
+authors:
+  - houssein
+date: 2018-11-05
+description: |
+  Реестры вроде npm изменили экосистему JavaScript к лучшему: они позволяют легко скачать и использовать более полумиллиона общедоступных пакетов. Но часто мы добавляем библиотеки, которые используем не полностью. Поэтому нужно анализировать пакеты и находить неиспользуемый код.
+codelabs:
+  - codelab-remove-unused-code
+tags:
+  - performance
+---
+
+Реестры вроде [npm](https://docs.npmjs.com/getting-started/what-is-npm) изменили
+экосистему JavaScript к лучшему: они позволяют легко скачать и использовать
+более _полумиллиона_ общедоступных пакетов. Но часто мы добавляем библиотеки,
+которые используем не полностью. Поэтому нужно **анализировать пакеты**
+и находить неиспользуемый код, а затем удалять **неиспользуемые** и **ненужные** библиотеки.
+
+## Анализ пакета
+
+С помощью DevTools можно легко узнать размер сетевых запросов:
+{% Instruction 'devtools-network', 'ol' %}
+{% Instruction 'disable-cache', 'ol' %}
+{% Instruction 'reload-page', 'ol' %}
```

Comment:
This instruction is not translated.

Reply:
It looks like it's a `{% code %}` part. "Instruction" doesn't appear on the [live page](https://web.dev/remove-unused-code/). What I see on the page is a list of instructions not included in the text of the article (they are included by the `{% Instruction … %}` code, I suppose):
![image](https://user-images.githubusercontent.com/220692/144850594-d11d4874-d87b-4857-bda4-f62b022da0c8.png)

Or am I understanding something wrong?

Reply:
Did a quick check on the code. Created a feature request for that. #6982. Could you please help to provide the translation here in comment? I'll see if I have time to implement that or wait for the team to do it.

Reply:
@chybie Sure. Here is the source I've compiled from the Instructions.js:

```
1. Press `Control+Shift+J` (or `Command+Option+J` on Mac) to open DevTools.
2. Click the **Network** tab.
3. Select the **Disable cache** checkbox.
4. Reload the page.
```

Here is my translation for this source:

```
1. Откройте DevTools, нажав `Control+Shift+J` (или `Command+Option+J`, если у вас Mac).
2. Перейдите на вкладку **Сеть** (Network).
3. Установите флажок **Отключить кеш** (Disable cache).
4. Перезагрузите страницу.
```

Reply:
Thanks @klyok. Created a PR #6984  . Once the review approved, we can rebase and it will be translated.